### PR TITLE
[MIRROR] Removes separated chemicals trait

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/42386
----------------------------
:cl:
del: removed the separated chemicals trait
/:cl:

I've been meaning to do this for a while. Separated chemicals is mainly used for botany explosives, which are a spammable one hit kill on almost everything. This has been especially bad since I fixed the bug that broke its interaction with squashing.

Botany will still have many tools without these instakills, especially after #42268. This should make botany's weapons both more balanced and more varied. 

Smokeplants will still be possible, but will require mutating traits into plants. 
In summary, botany needs a nerf in general, and this seems a good place to hit it.
note: EMPs will be lost as well

~~disclaimer: my children ded~~

### CODE REVIEW

**Comment by RafRoq** [comment]
Please don't

**Comment by yoyobatty** [comment]
Well, it's been reverted via modularization, we good to merge?

**Comment by RafRoq** [comment]
Thanks all fine then

**Review by yoyobatty — APPROVED** [review]
